Installation Guide

```diff
@@ -0,0 +1,266 @@
+## Installation Promgen On CentOS 7 (Updated - 11 Jan 2018)
+This documentation outlines the steps required to install, configure and run promgen application on CentOS 7.4 minimal server.
+
+### **Section 1 -  System Components Installation**
+
+#### Step 1: Install the IUS repository
+The IUS repository provides newer versions of some software in the official CentOS and Red Hat repositories. The IUS repository depends on the EPEL repository.
+```
+# yum -y install https://centos7.iuscommunity.org/ius-release.rpm
+```
+
+#### Step 2: Installation of  Git, Python 3.6 and All Requirements
+```
+# yum install python36u python36u-devel python36u-setuptools git gcc libxml2-devel libxslt-devel libffi-devel graphviz openssl-devel redhat-rpm-config -y
+```
+
+#### **Step 3: Install PIP**
+```
+easy_install-3.6 pip
```

Comment:
Is there a reason why you use easy_install here? While the pip version installed by rpm is sometimes a bit older, typically it's best to install as much as possible using the system packages when you can

Reply:
I think this is by habit, will retest the entire installation without the use of easy_install and check the outcome